locksmith/src/routes/v2/verifier.ts

```diff
@@ -20,6 +20,12 @@ router.put(
   (req, res) => verifierController.addVerifier(req, res)
 )
 
+router.put(
```

Comment:
Let's just reuse the previous route with an optional `verifierName` in the body.

Reply:
Ok, I will do asap